CarController.java

```diff
@@ -0,0 +1,56 @@
+package edu.ucmo.spring_example.controller;
+
+import edu.ucmo.spring_example.model.Car;
+import edu.ucmo.spring_example.dao.CarDao;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("cars")
+public class CarController {
+
+    @Autowired
+    private CarDao carDao;
```

Comment:
Should this really be private?

Reply:
Your right it should be protected instead.